interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -0,0 +1,48 @@
+<div ng-if="!queryStatus">
+  <h4>
+    <b>Experiments with the highest range: </b>{{experiments.length}}
+  </h4>
+  <button class="btn btn-primary" ng-click="showTop()"
+          ng-hide="topMode">Show Top {{topNum}}</button>
+  <button class="btn btn-primary" ng-click="showAll()"
+          ng-show="topMode">Show All</button>
+
+  <br />
+  (Click
+  <i class="fa fa-plus-square fa-lg" aria-hidden="true"></i> or
```

Comment:
I changed the button's background to gray, and removed the number of experiments. I still like the two "+/-" icons more than the real buttons. The two buttons are taking larger space and kind of shift user's attention. I give "show all" button a color that is more obvious because I think it is important.